spec/user_spec.rb

```diff
@@ -1,8 +1,11 @@
 describe User do
   it 'can create a new user' do
-    User.create
+    User.create(name: "name")
     expect(User.count).not_to eq 0
   end
 
-  # Write a new unit test here.
+  describe 'database table' do
+    it { is_expected.to have_db_column :name }
```

Comment:
I see that you added a validation for the presence of `name`, yet there is no test for it.

Reply:
I will add it